apps/test/unit/code-studio/components/BubbleChoiceTest.js

```diff
@@ -40,7 +40,12 @@ describe('BubbleChoice', () => {
     const wrapper = mount(<BubbleChoice {...DEFAULT_PROPS} />);
     const thumbnails = wrapper.find('img');
     assert.equal(1, thumbnails.length);
-    assert(thumbnails.at(0).node.src.includes(fakeSublevels[0].thumbnail_url));
+    assert(
+      thumbnails
+        .at(0)
+        .getDOMNode()
+        .src.includes(fakeSublevels[0].thumbnail_url)
+    );
```

Comment:
Root cause: [Private property `node` has been removed.](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#private-properties-and-methods-have-been-removed)